src/aggregate.d

```diff
@@ -126,7 +126,7 @@ public:
             sc2.inunion = 1;
         sc2.protection = Prot(PROTpublic);
         sc2.explicitProtection = 0;
-        sc2.structalign = STRUCTALIGN_DEFAULT;
+        sc2.aligndecl = null;
```

Comment:
I don't agree. `structalign` does not contain the word `value`. Your change is a minor preference.